src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -144,26 +176,52 @@ else if (module instanceof TreeWalkerFilter) {
         }
     }
 
+    /**
+     * {@inheritDoc} Processes the file.
```

Comment:
Why is more needed than the inherited doc ? I don't see us needing the sentence.

Reply:
there was a tool complained about `{@inheritDoc}` only. I don't remember what was it I think inspection

Reply:
https://circleci-tasks-prod.s3.us-east-1.amazonaws.com/storage/artifacts/2233fdd7-79cb-481c-a64c-0df373c0d898/239380ca-0213-4293-a3de-7b3eb22ce630/0/home/circleci/project/target/inspection-results/UnnecessaryInheritDoc.xml?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQVFQINEOHILZ4HBZ%2F20240709%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240709T190014Z&X-Amz-Expires=60&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEGsaCXVzLWVhc3QtMSJHMEUCIGuos9ECRyKz%2FEI2eNcxiS98o1IFtAK8h3Xm14%2BigOm%2FAiEAliysTESi6If2MS6%2BAUsesdx4brtggV466%2FgZXWy5yRYqqwIINBADGgwwNDU0NjY4MDY1NTYiDGP5fXW0FaUNa2yHwiqIAsfVLtQ53oLNZVj1yNciONwk6b82LnyXosvv%2Bku%2BF6IMyXm1LlaOK7s9GqoCKABVWvFEjEGdpTY%2Buom2Bh7TpZ4s7f04Ji24UxVYn12h2CG4TDMjxuBFWtfgZcpznK5V6A5YgdyVKJyrhiQqnfCcYsuu7KULsbA7p1CpbfLe7q53EHYwivqdE%2FDZaC0AwK4BXDhBflCi2ZvAmvy51fmR%2FHUnATY8bJSdHskcDrlpOy7kr1hKZigUvZhn%2ByTYnbWMh%2BPYdUuroycI8IaIZE9FgTjhQtPfr9ndaqwqifv2R8RKLhpJpGz5SIRHAYipyw3OqFbyDwk1Ad4MZAlY%2FRz%2FqxIGgbTBxK8j6jChjLa0BjqdAcqC1Sx2d%2FQ4k3kujP5K7X%2FePtqFOISS642J1JHEkgzwNfDjh8rz1kCTvXvZ7vCK3yTgNhX7PM5yiJf%2FPV0O9oaZP9oabfT%2FpZnPd6G%2FZcZvaKNQMPlZ4LcPuDerYBKVWF6V4LWx5A%2FCQeWAg7DACwXpgo4pqdxo2nbUHupq%2BpKHetce9RLuqdzAouXxDI8wcw9N9R0jfyfkPsEIxuY%3D&X-Amz-SignedHeaders=host&x-id=GetObject&X-Amz-Signature=0aa4e1223c40b2506e8de05e06307a0f1f6c21f0ce29ea6f66157e807413e004

This seems like a false violation since we added the suppressions. I don't know the intellij enough to know if we can suppress this without a javadoc. Please restore the sentence.

Reply:
restored